### PR TITLE
Feature/#1699

### LINKED ISSUE #1699: Add banner which explains user how to access old Mainnet markets
With the new xDai tight integration old mainnet markets will not be the default markets on omen.eth. For that we need to explain the user how to get full access/old experience back if they need to. 

desktop

banner

Note: 
- Can be removed from the user and we should save it on localStorage.
- Learn more should be a hyperlink (link of blogpost not ready)

Figma:
[Design Specification](https://www.figma.com/file/aTivxuHmp2l6Sn8nho1Ngu/Playground?node-id=2229%3A1)

### PR DESCRIPTION
**Pull request by Mi-Lan**
closes #1699 
- Link is still missing in learn more!
- Also we should maybe change it a little for mobile?

### CODE REVIEW

File: app/src/components/market/sections/market_list/img/xDaiIntegration.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<svg width="52" height="52" viewBox="0 0 52 52" fill="none" xmlns="http://www.w3.org/2000/svg">
```
[the remark this answers is not in the document]
Comment (Mi-Lan) [reply app/src/components/market/sections/market_list/img/xDaiIntegration.svg:1]: yeah i can add it my thinking was that it wasnt going to be reused that much or change sizes...And it doesn't fit description of icon like the rest...

[the remark this answers is not in the document]
Comment (KadenZipfel) [reply app/src/components/market/sections/market_list/img/xDaiIntegration.svg:1]: Yeah, I think you're right, let's keep it as is

**Review by KadenZipfel — APPROVED** [review]

**Review by hexyls — APPROVED** [review]
Looks good! 

Like you said maybe we should add some extra top margin on mobile?

Screen Shot 2021-03-08 at 3 25 58 pm

**Comment by Mi-Lan** [comment]
> Looks good!
> 
> Like you said maybe we should add some extra top margin on mobile?
> 
> Screen Shot 2021-03-08 at 3 25 58 pm

Thanks! Yeah maybe @pimato has some idea?

**Comment by pimato** [comment]
increase the horizontal padding from 20px to 24px:
Bildschirmfoto 2021-03-08 um 18 49 03

I changed my design accordingly.

**Review by pimato — CHANGES_REQUESTED** [review]
Bildschirmfoto 2021-03-08 um 18 50 24

Mouse hand hover should only work for the hyperlink and the close icon!

**Comment by pimato** [comment]
learn more should have default hover styling

**Comment by pimato** [comment]
close icon hover styling should be identical to the connection modal icon:
Bildschirmfoto 2021-03-08 um 18 52 31

**Comment by Mi-Lan** [comment]
> Bildschirmfoto 2021-03-08 um 18 54 49
> 
> text should have additional styling:
> 
> ```
> letter-spacing: 0.2px;
> line-height: 19px;
> ```

i fixed just the line-height letter spacing was on point...

**Review by pRoy24 — APPROVED** [review]
LGTM

**Review by pimato — APPROVED** [review]

**Pull request merged**